dll/win32/shdocvw/winlist.cpp

```diff
@@ -69,6 +82,367 @@ VariantClearLazy(_Inout_ LPVARIANTARG pvarg)
     return S_OK;
 }
 
+static INT
+GetSharedServiceMode(
```

Comment:
An explanation about what these two functions are for, would be welcome!

Reply:
To be continued...

Reply:
Added code.